src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -162,5 +162,10 @@ public int MaxModelValidationErrors
         /// Gets or sets the default value for the Permanent property of <see cref="RequireHttpsAttribute"/>.
         /// </summary>
         public bool RequireHttpsPermanent { get; set; }
+
+        /// <summary>
+        /// Gets or sets the flag to allow binding undefined values to Enum types.
+        /// </summary>
+        public bool AllowBindingUndefinedValueToEnumType { get; set; } = true;
```

Comment:
Usually you try to find a way to make the default value false. It's always non-obvious when the default is a true.

Did you discuss this with @Eilon - I was under the impression we wanted the **new** behavior as the default.